src/KeyboardInputModal.scss

```diff
@@ -18,6 +9,15 @@
     flex-direction: column;
     justify-content: center;
     margin-top: 1rem;
+
+    .modal {
```

Comment:
.modal is not a child of .KeyboardInputModal__content, so this rule is not applying anything, we can remove it.

Reply:
Good catch, and done.